corehq/apps/hqwebapp/templates/hqwebapp/tables/bootstrap5_htmx.html

```diff
@@ -0,0 +1,78 @@
+{% extends "hqwebapp/tables/bootstrap5.html" %}
+{% load i18n %}
+{% load django_tables2 %}
+{% load hq_shared_tags %}
+{% load hq_tables_tags %}
+
+{% block table.thead %}
+  {% if table.show_header %}
+    {% render_header 'htmx' %}
+  {% endif %}
+{% endblock table.thead %}
+
+{% block table-container-attrs %}
+  hx-get="{{ request.path_info }}{% querystring %}"
+  hx-replace-url="{% querystring %}"
+  hx-swap="outerHTML"
+  hx-trigger="hqRefresh"
+  hq-hx-loading="{{ table.loading_indicator_id  }}"
+{% endblock %}
+
+{% block after_table %}
+  <div
+    class="table-loading-indicator"
+    {% if table.loading_indicator_id %}
+      id="{{ table.loading_indicator_id  }}"
+    {% endif %}
+  >
+    <div class="spinner-border" role="status">
+      <span class="visually-hidden">{% trans "Loading..." %}</span>
+    </div>
+  </div>
+{% endblock %}
+
+{% block select-per-page-attr %}
+  name="{{ table.per_page_field }}"
+  hx-get="{{ request.path_info }}"
+  hx-target="{% if table.css_id %}#{{ table.css_id }}{% else %}div.table-container{% endif %}"
+  hx-swap="outerHTML"
+  hq-hx-loading="{{ table.loading_indicator_id  }}"
```

Comment:
I like the shortened name, but I feel it abstracts the `hx-` (for `HTMX`) connection a little too much. It's clear from `hq-hx-` that this is a custom HQ HTMX attribute. And since Alpine attributes begin with `x-`, `hqx-` seems more ambiguous as it could reference alpine, not `HTMX`. I would prefer to keep the naming scheme as is. Regardless, I feel such a change would be out of scope for this PR given that there are other extensions existing that already a part of HQ.